docs/Silver Blaze.html

```diff
@@ -6,6 +6,7 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<title>
 		Silver Blaze
+
 	</title>
```

Comment:
If this is generated by running the tool, there should not be an empty line here